source/AI.cpp

```diff
@@ -2374,13 +2385,7 @@ void AI::AutoFire(const Ship &ship, Command &command, bool secondary) const
 	bool beFrugal = (ship.IsYours() && !escortsUseAmmo);
 	if(person.IsFrugal() || (ship.IsYours() && escortsAreFrugal && escortsUseAmmo))
 	{
```

Comment:
You can probably modify the above lines, but at minimum you can drop the parenthesis here now since there is only one statement following the `if` branch.